<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="learningList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>314</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="masteredLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="learningLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="detailBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string>详  细  解  释</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>返  回</string>
    </property>
   </widget>
   <widget class="QListWidget" name="masteredList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>52</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="transBrowser">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>52</y>
      <width>331</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="relearnBtn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>221</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>重  新  学  习</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>601</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame</zorder>
   <zorder>learningList</zorder>
   <zorder>masteredLabel</zorder>
   <zorder>learningLabel</zorder>
   <zorder>detailBtn</zorder>
   <zorder>exitBtn</zorder>
   <zorder>masteredList</zorder>
   <zorder>transBrowser</zorder>
   <zorder>relearnBtn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitBtn</sender>
   <signal>clicked()</signal>
   <receiver>UserWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>509</y>
    </hint>
    <hint type="destinationlabel">
     <x>743</x>
     <y>530</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
